src/components/shared/Slider/Label.tsx

```diff
@@ -1,46 +1,56 @@
-import { Animated } from 'react-native';
+import { Animated, StyleProp, StyleSheet, TextStyle, ViewStyle } from 'react-native';
+
 import React from 'react';
 import styled from 'styled-components/native';
 
+interface LabelProps {
```

Comment:
`labelStyle` and `textStyle` should be in  `Props` interface without `LabelProps`.
And, `labelStyle` -> `style` is better for the name as the default props name is `style`.